wifiphisher/common/interfaces.py

```diff
@@ -346,6 +371,7 @@ def __init__(self):
 
         self._name_to_object = dict()
         self._active = set()
+        self._internet_interface = ''
```

Comment:
Since wireless card can be used as as internet card now, I have to remember what card have been used as internet card; otherwise we may choose this internet card as deauth/ap adapter